Tools/autotest/common.py

```diff
@@ -4851,6 +4857,7 @@ def test_dataflash_over_mavlink(self):
             self.mavproxy.send("dataflash_logger set verbose 0\n")
             self.delay_sim_time(1)
             self.drain_mav() # hopefully draining COMMAND_ACK from that failed arm
+            self.zero_throttle()
```

Comment:
Not happy with sprinkling these before every arm call.

It adds cruft which we don't necessarily need.

Do we really need these everywhere?

Reply:
I removed them all; will try to think of a better way to do it